docs/package.json

```diff
@@ -23,7 +17,11 @@
     "test:ember": "ember test",
     "test:all": "ember try:each"
   },
+  "dependencies": {
+    "ember-sortable": "workspace:../addon"
```

Comment:
this explicitly declares which ember-sortable we want to use (always local workspace)